lib/redlock/client.rb

```diff
@@ -148,13 +148,13 @@ def valid_lock?(lock_info)
 
     class RedisInstance
       module ConnectionPoolLike
-        def with
+        def run_with
           yield self
         end
```

Comment:
Do we still need to offer `with`? Is there any chance of breaking something?

```ruby
        def with
          yield self
        end

        def run_with
          yield self
        end
```

Reply:
its broken if we keep it. I think it should be just renamed. but the name might not be your favorite..

Reply:
Thank you @mathieujobin